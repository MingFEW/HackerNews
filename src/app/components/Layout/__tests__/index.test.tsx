import { createRenderer } from 'react-test-renderer/shallow'

import { PageWrapper } from '../index'

const shallowRenderer = createRenderer()

describe('<PageWrapper />', () => {
  it('should render and match the snapshot', () => {
    shallowRenderer.render(<PageWrapper>Content</PageWrapper>)
    const renderedOutput = shallowRenderer.getRenderOutput()
    expect(renderedOutput).toMatchSnapshot()
  })
})
