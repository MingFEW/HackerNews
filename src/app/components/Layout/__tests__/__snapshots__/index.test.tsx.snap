// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageWrapper /> should render and match the snapshot 1`] = `
<div
  tw="relative w-full h-full min-h-screen bg-background"
>
  <Header />
  <div
    tw="w-full pt-20 px-6 mx-auto md:max-w-[640px] md:px-0 lg:max-w-[768px] relative z-[1]"
  >
    Content
  </div>
  <div
    className="site-background"
  >
    <div
      tw="z-0 block fixed top-0 bottom-0 left-0 inset-x-0 bg-gradient-to-b from-darkJungleGreen"
    />
    <div
      tw="fixed inset-x-0 top-0 bottom-0 left-0 z-0 bg-top bg-no-repeat  [background-image: url(/images/bg_overlayy.png)] [background-size: 136.75rem 100%] h-[52.625em]"
    />
  </div>
</div>
`;
